roles/ceph-common/tasks/main.yml

```diff
@@ -64,24 +64,16 @@
   tags:
     - package-install
 
-- name: get ceph rhcs version
-  shell: rpm -q --qf "%{version}\n" ceph-common | cut -f1,2 -d '.'
+- name: get ceph version
+  shell: ceph --version | cut -f3 -d ' '
```

Comment:
The latest approach looks good to me.
Do you mind modifying the other part using ceph_version too?

I think we need to remove this one: https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-mon/tasks/start_monitor.yml#L57-L61 and update those two https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-mon/tasks/create_mds_filesystems.yml#L12 and https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-mon/tasks/create_mds_filesystems.yml#L17 with `0.80.0`.
Finally those https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-mon/tasks/secure_cluster.yml#L5 and https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-mon/tasks/secure_cluster.yml#L12 with `0.94.0`.

Thanks!